### DISCUSSION TITLE
Request to add Smart Connection feature.

### DISCUSSION THREAD

**Discussion by woxin667** [toeverything/AFFiNE#7205] [Ideas]
Would the development team consider adding a feature similar to the Obsidian Smart Connections plugin in the future? It allows users to discover the associations between notes.

https://github.com/brianpetro/obsidian-smart-connections

**Comment by HeJiachen-PM** [comment]
Good idea! I think it can be a nice to have feature in Affine AI. However, I would love it more if we can also have such a plugin.

**Reply by woxin667** [reply]
Thanks for the recognition! But unfortunately, I don't have the ability to develop.
